tests/src/test/scala/cats/tests/OneAndTests.scala

```diff
@@ -180,6 +174,13 @@ class OneAndTests extends CatsSuite {
       got should === (expected)
     }
   }
+
+  test("filter includes elements based on a predicate") {
+    forAll { (nes: NonEmptyStream[Int]) =>
+      nes.filter(_ % 2 == 0) should ===(nes.unwrap.filter(_ % 2 == 0))
+    }
+  }
+
```

Comment:
I don't quite know what you mean. The original test in this case is `MonadCombineTests`, which has been removed, but I did needlessly use `_ % 2 == 0` instead of an arbitrary function; that's been corrected now.